terraform/020-database/db-users.sql

```diff
@@ -0,0 +1,34 @@
+Once phpmyadmin is running at <idp>-pma.<domain>, execute the following
+sql statements to create users and databases ...
```

Comment:
Should this be a comment?

Reply:
Yes, technically, it should. I just restored it as it was.